vnext/PropertySheets/External/Microsoft.ReactNative.WinAppSDK.props

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- 
+  Copyright (c) Microsoft Corporation.
+  Licensed under the MIT License.
+
+  This file will be consumed by ALL WindowsAppSDK projects (both inside
+  and outside of this repo) that build on top of Microsoft.ReactNative.
+  Do not make any changes here unless it applies to ALL such projects.
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- Keeping this at the top of the imports ensures we can use the autolinking 
+       process to set properties without having to edit the project file -->
+  <Import Project="$(ProjectDir)\AutolinkedNativeModules.g.props"
+          Condition="Exists('$(ProjectDir)\AutolinkedNativeModules.g.props')" />
+
+  <PropertyGroup>
+    <ReactNativeCodeGenEnabled Condition="'$(ReactNativeCodeGenEnabled)' == ''">true</ReactNativeCodeGenEnabled>
+  </PropertyGroup>
+
+  <Import Project="$(MSBuildThisFileDirectory)Microsoft.ReactNative.Uwp.Common.props" />
```

Comment:
weird that we are including the uwp common props in winappsdk. Maybe we should rename the uwp common props file to something else?

Reply:
looks like uwp common props just import react native common props, I'll switch it out for that.

Reply:
For now, I added a separate file for WinAppSDK.common props and WinAppSDK.CSharpApp.props but they're pretty identical to the UWP ones, I'm honestly leaning towards renaming all of the props files to remove the "uwp" specifier to avoid duplicate files, then if they diverge down the road, we can split them into separate props files. @NickGerleman @jonthysell, thoughts?